source/OTRServer/public/display_render.js

```diff
@@ -0,0 +1,275 @@
+// ====================================================================================
+// INIT
+// ====================================================================================
+// Canvas Initialization
+let cnv = document.getElementById("canvas-parent");
+// This is old style initialization apparently.
```

Comment:
So we don't need this?

Reply:
Yeah we keep this. 
The documentation for the library is all wack and the latest distribution is running based on what the old documentation is saying, not the new documentation.
It is really awful and it keeps popping up in other areas.